can/interfaces/vector/canlib.py

```diff
@@ -515,6 +516,15 @@ def handle_canfd_event(self, event: xlclass.XLcanRxEvent) -> None:
     def send(self, msg: Message, timeout: typing.Optional[float] = None):
         self._send_sequence([msg])
 
+    @property
+    def bitrate(self) -> int:
+        return self.bitrate
```

Comment:
Doesn't this recuse infinitely? Usually, I see this being implemented with a private attribute `self._bitrate`.

Reply:
oops yeah I'll get that fixed. Just tossed this together quick to get some feedback.